### PR TITLE
Allow puppetlabs/stdlib 6.x

### PR DESCRIPTION
**Pull request by pillarsdotnet**
* Change [`metadata.json`](https://github.com/voxpupuli/puppet-snmp/blob/master/metadata.json#L18) to allow [`puppetlabs/stdlib 6.x`](https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/metadata.json#L3).

### CODE REVIEW

File: manifests/client.pp [hunk 1]
```diff
@@ -49,13 +49,11 @@
...
   }
 
-  if $facts['os']['family'] != 'Suse' {
-    if $package_name != undef {
-      package { 'snmp-client':
-        ensure => $package_ensure,
-        name   => $package_name,
-        before => File['snmp.conf'],
-      }
+  if ($package_name != undef) and ($package_name != $snmp::package_name) {
```
Comment (bastelfreak) [inline manifests/client.pp]: this can be simplified:
```suggestion
  if ($package_name) and ($package_name != $snmp::package_name) {
```

Comment (Dan33l) [inline manifests/client.pp]: Can move this in a dedicated PR with a clear title ?

Comment (pillarsdotnet) [reply manifests/client.pp]: No; I believe the title and explanation are clear.
